pages/key-value-pairs/screenshot.page.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import KeyValuePairs from '~components/key-value-pairs';
+import ScreenshotArea from '../utils/screenshot-area';
+
+export default function () {
+  return (
+    <article>
+      <h1>KeyValuePairs for screenshot</h1>
```

Comment:
Minor: Let's rename this file to "simple.page.tsx" to be consistent with the rest of the dev pages.

Reply:
updated